bigtable/admin.go

```diff
@@ -269,6 +276,10 @@ func (ac *AdminClient) CreateTableFromConf(ctx context.Context, conf *TableConf)
 	} else if conf.DeletionProtection == Unprotected {
 		tbl.DeletionProtection = false
 	}
+	if conf.ChangeStreamRetention != nil && conf.ChangeStreamRetention.(time.Duration) != 0 {
```

Comment:
Do we need to check the retention is between 1 and 7?

Reply:
Seems like we should leave that up to server side validation https://google.aip.dev/client-libraries/4223